UnityAds/src/main/java/com/mopub/mobileads/UnityRewardedVideo.java

```diff
@@ -161,28 +162,16 @@ public void show() {
             return;
         }
 
-        if (UnityAds.isReady(mPlacementId)) {
-            // Lets Unity Ads know when ads succeeds to show
-            MediationMetaData metadata = new MediationMetaData(mLauncherActivity);
-            metadata.setOrdinal(++impressionOrdinal);
-            metadata.commit();
-
-            UnityAds.show(mLauncherActivity, mPlacementId);
-        } else {
-            // Lets Unity Ads know when ads fail to show
-            MediationMetaData metadata = new MediationMetaData(mLauncherActivity);
-            metadata.setMissedImpressionOrdinal(++missedImpressionOrdinal);
-            metadata.commit();
+        if (mLoadedPlacementId ==  null) {
```

Comment:
Same comment here for null checking

Reply:
So i have to distinguish between when the ad was not loaded and when the actual placement id is null.  That is what this check is doing.  In the event its null, i use the created placement id (what we would have attempted to load with) as the placement id.  in the event that the user supplied a null placement mPlacementId would also be null and that would be sent to webview.

Reply:
Same as [here](https://github.com/Applifier/mopub-android-mediation/pull/6#discussion_r601785624)

Reply:
This also differs from the logic in the iOS adapter:  https://github.com/Applifier/mopub-ios-mediation/blob/b79b0c60606adc95bbb920a1e86293ee1ddf5b2f/UnityAds/UnityAdsRewardedVideoCustomEvent.m#L104